src/infi/clickhouse_orm/fields.py

```diff
@@ -16,19 +16,21 @@ class Field(object):
     class_default = 0
     db_type = None
 
-    def __init__(self, default=None, alias=None, materialized=None):
+    def __init__(self, default=None, alias=None, materialized=None, readonly=None):
```

Comment:
Some of the Field subclasses override __init__, so you should add the readonly argument there too

Reply:
Fixed.